### DISCUSSION TITLE
ui.select with stack-label prop

### DISCUSSION THREAD

**Discussion by MateoSaezMata** [zauberzeug/nicegui#3808] [Q&A] [answered]
### Description

I'm trying to set a "placeholder" (ie `stack-label` prop) to my `ui.select` element.

My code:
```python
from nicegui import ui

ui.select(['First', 'Second'], label="Please select").props('stack-label')

ui.run()
```

Result:
(before selection)
![image](https://github.com/user-attachments/assets/19e564e0-14af-41bd-8913-2cfc6c3cbe0f)

(after selection)
![image](https://github.com/user-attachments/assets/1d6be3bb-9559-4d28-87b1-e73476ecd31e)

It will be nice if the `ui.select` can display some placeholder like `ui.input` element like this:
![image](https://github.com/user-attachments/assets/04a77df6-c510-48ca-bf02-5f6474046800)

Am I missing something?

Thanks in advance!

**Comment by falkoschindler** [comment, marked as answer]
Hi @MateoSaezMata,

It looks like the `ui.select` element is too narrow to actually see the stacked label. This works:
```py
ui.select(['First', 'Second'], label="Please select").props('stack-label').classes('w-40')
```
<img width="178" alt="Screenshot 2024-09-29 at 15 46 16" src="https://github.com/user-attachments/assets/c1b580f0-bde5-453e-9245-719c92e1d0ad">

**Reply by MateoSaezMata** [reply]
Thanks!!
That's what I wanted.

Amazing framework btw
